crates/nu-protocol/src/value/mod.rs

```diff
@@ -12,7 +12,7 @@ use crate::engine::EngineState;
 use crate::ShellError;
 use crate::{did_you_mean, BlockId, Config, Span, Spanned, Type, VarId};
 use byte_unit::ByteUnit;
-use chrono::{DateTime, Duration, FixedOffset};
+use chrono::{DateTime, Datelike, Duration, FixedOffset};
```

Comment:
do we need `chrono::DateLike` to call `.to_rfc3339`?

Reply:
No, but we do need it to call `.year()`.

Reply:
ooooh i see, thanks for the explaination :+1: